src/custom_heroes.opy

```diff
@@ -7,6 +7,7 @@
 
 #!include "heroes/ana.opy"
 #!include "heroes/brigitte.opy"
+#!include "heroes/doomfist.opy"
```

Comment:
Same comment as before

The imports should be grouped by tank, damage, and support heroes (+ alphabetic for each group).

Reply:
Yes, ill do that